node/actors/network/src/gossip/runner.rs

```diff
@@ -107,23 +86,23 @@ impl rpc::Handler<rpc::push_block_store_state::Rpc> for &PushBlockStoreStateServ
         req: rpc::push_block_store_state::Req,
     ) -> anyhow::Result<()> {
         req.0.verify(self.net.genesis())?;
-        self.state.send_replace(req.0);
+        self.blocks.send_replace(req.0);
         Ok(())
     }
 }
 
 #[async_trait]
-impl rpc::Handler<rpc::push_batch_store_state::Rpc> for &PushBatchStoreStateServer {
+impl rpc::Handler<rpc::push_batch_store_state::Rpc> for &PushServer<'_> {
     fn max_req_size(&self) -> usize {
-        self.max_batch_size.saturating_add(kB)
+        10 * kB
     }
```

Comment:
I notice you removed this from the server side but not from the [client side](https://github.com/matter-labs/era-consensus/blob/1afde79e3891c14b080d35b3f69593843fc54970/node/actors/network/src/gossip/runner.rs#L338). 

Without this change the tests in `zksync-era` failed as the payload size was larger than 10kB, and after this change the gossip of batches was actually enabled. Am I missing something that keeps requests small now?

Reply:
There are many different messages here:
* PushBatchStoreState request: just range of BatchNumbers (begin and end)
* PushBAtchStoreState response: empty
* GetBatch request: just BatchNumber
* GetBatch response: the actual large batch.

Now that you mention it, I see that The PushBatchStoreState request is incorrectly defined. Let me fix it.

Reply:
done